rotkehlchen/chain/optimism/transactions.py

```diff
@@ -7,6 +7,7 @@
 
 if TYPE_CHECKING:
     from rotkehlchen.db.dbhandler import DBHandler
+
```

Comment:
This is from isort (probably because it is a relative import)

Reply:
Even if it is from isort this file **does not need to be touched**. Undo it. No isort configuration changed.

If this file needed to change due to our isort config all linting runs of the CI in develop would fail.

 If it's your own local editor that does this then configure your editor.